starter-code/css/main.css

```diff
@@ -1,8 +1,123 @@
-/*
-Colors:
+body {
+    font-family: ‘Lato’, sans-serif;
+}
+.navbar {
+    border: indigo;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
 
-Text: 1A1A1A
-Green: #00B172
-White: #FFF
+ul {
+    display: flex;
+    list-style: none;
+}
 
-*/
+ul > li > a {
+    padding: 10px;
+    text-decoration: none;
+    color: black;
+}
+
+.logo {
+    display: flex;
+    align-items: center;
+}
+
+.logo img {
+    width: 150px;
+    height: auto;
+}
+
+.header {
+    background-image: url("../images/landing.jpg");
+    background-size: contain;
+    height: 100vh;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+}
+
+.header p {
+    font-size: 2em;
+}
+
+.header h1 {
+    font-size: 5em;
+}
+
+.benefits{
+    text-align: center;
+    font-size: 2em;
+    margin-bottom: 100px;
+}
+
+.container {
+    display: flex;
+    justify-content:center;
+    align-items: center;
+}
+
+.container h3 {
+    color: #00B172;
+}
+
+.perks {
+    width: 500px;
+    padding: 10px;
+    justify-content: baseline;
+    align-content: center;
+}
+
+.perks img {
+    width: 150px;
+    height: 120px;
+}
+
+hr {
```

Comment:
Avoid using the html tag selector, and always try and using classes or identifiers instead as it is best practice that way :)